src/components/input/input.ts

```diff
@@ -75,10 +78,22 @@ import { Platform } from '../../platform/platform';
 @Component({
   selector: 'ion-input',
   template:
-    '<input [type]="type" [(ngModel)]="_value" (blur)="inputBlurred($event)" (focus)="inputFocused($event)" [placeholder]="placeholder" class="text-input" [ngClass]="\'text-input-\' + _mode">' +
+    '<input [type]="type" [(ngModel)]="_value" (blur)="inputBlurred($event)" ' +
+    '(focus)="inputFocused($event)" [placeholder]="placeholder" ' +
+    'class="text-input" [ngClass]="\'text-input-\' + _mode" ' +
+    '[attr.autocomplete]="autocomplete" [attr.autocorrect]="autocorrect"' +
+    '[attr.autocapitalize]="autocapitalize" [autofocus]="autofocus"' +
+    '[attr.list]="list" [attr.max]="max" [attr.maxlength]="maxlength"' +
+    '[attr.min]="min" [attr.minlength]="minlength" [readonly]="readonly"' +
```

Comment:
1. why [readonly], [required], [spellcheck]? should not be [attr.SOMETHING]?
2. use the variable not the getter: `_readonly` instead of `readonly` etc.
3.  Please one line per attr
4. autofocus does not work with angular 2